doc/api/https.md

```diff
@@ -7,47 +7,53 @@ separate module.
 
 ## Class: https.Agent
 <!-- YAML
-added: v0.4.5
+added: v0.4.4
 -->
 
-An Agent object for HTTPS similar to [`http.Agent`][].  See [`https.request()`][]
-for more information.
+**Inherits:** [`http.Agent`]
+
+An Agent object for HTTPS.
 
 ## Class: https.Server
 <!-- YAML
 added: v0.3.4
 -->
 
-This class is a subclass of `tls.Server` and emits events same as
-[`http.Server`][]. See [`http.Server`][] for more information.
+**Inherits:** [`tls.Server`]
+**Implements:** [`http.Server`]
```

Comment:
There should be some explanation for this implements. Inherits and implements is kinda confusing.